emeritus-tech/landing-pages-libraries
# [CPT-433] Create stylesheets for KMS

🎟 [Jira Ticket](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-433)

🚢 Deployment Instructions

N/A

✅ QA instructions

* Run command npm run build to compile.
* Open dist/index.html on browser:
  * Design should look exactly as original [KMS](https://admissions.emeritus.org/programs/kellogg-executive-education/mastering-sales) landing page
  * Modals should work as expected (without the modals.js file will only function in fallback mode, meaning that only clicking the X button will close it and ESC key or clicking outside will not)

⚠️ Risks

Low

🖼 Previews

![screencapture-file-Users-holasoyandrea-Desktop-Emeritus-landing-pages-libraries-styles-dist-index-html-2020-04-24-14_27_59](https://user-images.githubusercontent.com/58177278/80241097-6725a680-8639-11ea-840c-5254c232dee6.png)

📝 Notes

*  As requested, hero form and modal form wasn't implemented for now.
*  Sticky nav will not show up (will not be broken tho) until the following new refactored JS file is implemented: sticky.js
*  Modals will not work entirely as expected unless original modals.js file is implemented.
